tests/system/Models/WhenWhenNotModelTest.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Models;
+
+use Tests\Support\Models\SecondaryModel;
+
+/**
+ * @group DatabaseLive
+ *
+ * @internal
+ */
+final class WhenWhenNotModelTest extends LiveModelTestCase
+{
+    public function testWhenWithSingleCallback(): void
```

Comment:
This method name is difficult to understand what is tested.
```suggestion
    public function testWhenWithTrueCondition(): void
```